source/includes/administration/_custom_ui_settings.md

```diff
@@ -0,0 +1,246 @@
+### Custom UI Settings
+
+<!-------------------- FIND CUSTOM UI SETTINGS -------------------->
+
+#### Find custom ui settings associated to an organization
+
+`GET /reseller/settings/custom_ui/find?organizationId=:id`
+
+Retrieve the custom ui settings associated to an organization. If the `organizationId` is omitted, the authenticated user's organization will be used.
+
+```shell
+# Retrieve the custom ui settings
+curl "https://cloudmc_endpoint/api/rest/reseller/settings/custom_ui/find?organizationId=10572c3d-16e5-450f-8af8-a01e50dc52d4" \
```

Comment:
i think this should be https://cloudmc_endpoint/api/v2/reseller/...

Reply:
you are correct... will change it

Reply:
28a54025d74ba155e7339ea0a8207d38ead6aea3